worker2.env.example

```diff
@@ -2,12 +2,15 @@
 # Warehouse Workers fetch jobs/payloads from a queue
 # There are many technologies that can provide a queue implementation: SQS, RabbitMQ, redis, etc.
 ##
+AWS_ACCESS_KEY_ID= <YOUR>
```

Comment:
change key & secret to minio default key & secret

Reply:
done !